documentation/docs/operation/keyboard-shortcuts.md

```diff
@@ -2,17 +2,55 @@
 
 | Keyboard Combo | Command | Description |
 |-------- | ----------- | ----------- |
-| `Double CTRL + C` | Exit | Press `CTRL + C` twice to exit the bot.
-| `CTRL + S` | Status | Show bot status.
-| `CTRL + F` | Search | Toggles search in log pane (see below for usage)
-| `CTRL + A` | Select All | Select all text in input pane [used for text edit in Input pane only].
-| `CTRL + Z` | Undo | Undo action in input pane [used for text edit in Input pane only].
-| `Single CTRL + C` | Copy | Copy text [used for text edit in Input pane only].
-| `CTRL + V` | Paste | Paste text [used for text edit in Input pane only].
-| `CTRL + X` | Exit Config | Exit from the current configuration question.
+| CTRL + C + C | Exit | Press CTRL + C twice to exit the bot.
+| CTRL + S | Status | Show bot status.
+| CTRL + F | Search | Toggles search in log pane (see below for usage)
+| CTRL + A | Select All | Select all text in input pane only.
+| CTRL + Z | Undo | Undo action in input pane only.
+| CTRL + X | Exit Config | Exit from the current configuration question.
 
 !!! tip "Tip: How to use the Search feature"
     1. Press `CTRL + F` to trigger display the search field
     2. Enter your search keyword (not case sensitive)
     3. Hit `Enter` to jump to the next matching keyword (incremental search)
-    4. When you are done. Press `CTRL + F` again to go back to reset.
\ No newline at end of file
+    4. When you are done. Press `CTRL + F` again to go back to reset.
+
+## Copy and Paste
+
+### Linux
+
+| Keyboard Combo | Command |
+|-------- | ----------- |
+| CTRL + C | Copy
+| SHIFT + RMB <img src="/assets/img/rmb.png" height="15"> | Paste
+
+To highlight, hold **SHIFT + LMB** <img src="/assets/img/lmb.svg" height="13"> and drag across the text you want to select.
+
+### macOS
+
+| Keyboard Combo | Command |
+|-------- | ----------- |
+| ⌘ + C | Copy
+| ⌘ + V | Paste
+
+To enable highlighting in Hummingbot installed via macOS binary dmg installer, disable mouse reporting by pressing **`⌘ + R`** or go to **View > Allow Mouse Reporting**.
```

Comment:
Use this note instead of the paragraph above

```
!!! note "Allow Mouse Reporting"
    To select text on macOS, you may need to enable the **Allow Mouse Reporting** option by pressing **`⌘ + R`** or selecting  **View > Allow Mouse Reporting** in the menu bar.

Reply:
@mifeng should I include the screenshot inside the note too or is this ok?

![image](https://user-images.githubusercontent.com/50150287/88449135-2119f200-ce77-11ea-8436-c9e54578c2ba.png)